cmd/migrate.go

```diff
@@ -44,28 +44,19 @@ func migrateCmd(cmd *cobra.Command, _ []string) error {
 	}
 	defer syncZapLogger(logger)
 
-	pgOpt, err := pgCfg.ToPGOptions()
+	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?sslmode=disable", pgCfg.User, pgCfg.Password, pgCfg.Host, pgCfg.Port, pgCfg.Database)
```

Comment:
We have PG Client Dialer implementation has a very advance behaviour which allow us to support many different SSL modes and it is already integrated with out cmd flags. Said that, I would rather investtime in trying to reuse that PG Client into this migration instance, most likely you will need to implement the Driver interface, but IMO we cannot get along without it in this PR

Reply:
golang-migrate already supports SSL mode. We only need to fill the corresponding fields. I will add SSL mode in a next PR:
https://github.com/golang-migrate/migrate/blob/master/database/postgres/README.md#use-in-your-go-project

Reply:
implemented in next PR